src/worker/move-contact-worker.ts

```diff
@@ -0,0 +1,160 @@
+import axios from 'axios';
+import { spawn } from 'child_process'; 
+import { Worker, Job } from 'bullmq';
+
+import Auth from '../lib/authentication';
+import { queuePrivateKey, redisConnection } from '../shared/queue-config';
+import { queueManager } from '../shared/queues';
+
+export interface MoveContactData {
+  parentId: string;
+  contactId: string;
+  sessionToken: string;
+  instanceUrl: string;
+}
+
+export type JobResult = { success: boolean; message: string };
+
+export class MoveContactWorker {
+  private readonly DELAY_IN_MILLIS = 360_000;   // 1 hour
+  private readonly MAX_TIMEOUT = 180_000;       // 30 minutes timeout
```

Comment:
These are going to be very awkward to try to recover if large moves ever crash or are aborted.  Can we make this longer - like 60 or 90 minutes? For now at least, I don't think we want to be terminating jobs as they process.